pandas/util/testing.py

```diff
@@ -673,7 +673,7 @@ def capture_stderr(f):
     AssertionError: assert 'foo\n' == 'bar\n'
     """
 
-    @wraps(f)
+    @compat.wraps(f)
```

Comment:
Looks like `wraps` wasn't preserving the signature in Py27 which would cause this to choke when combined with a parametrized function. Simply switched to the function from combat for now - providing as reference in case anyone runs into a similar issue in the future